### PR TITLE
release 0.2.1

### LINKED ISSUE #120: Release opencage 0.2.0
Prepare for release:

* [x] `devtools::build_readme()`
* [x] Check [current CRAN check results](https://cran.rstudio.org/web/checks/check_results_opencage.html) (just a note re vignette title, which have been rewritten)
* [x] `devtools::check(remote = TRUE, manual = TRUE)`
* [x] `devtools::check_win_devel()`
* [x] `rhub::check_for_cran()`
* [x] `revdepcheck::revdep_check(num_workers = 4)`
* [x] Update `cran-comments.md` (add comment re: `<<-`?)
* [x] [Polish NEWS](https://style.tidyverse.org/news.html#news-release)

Submit to CRAN:

* [x] email CRAN re. maintainer change
* [ ] Draft blog post (#56)
* [x] `usethis::use_version('minor')`
* [x] `devtools::submit_cran()`
* [x] Approve email

Wait for CRAN...

* [x] Accepted :tada:
* [x] `usethis::use_github_release()`
* [x] `usethis::use_dev_version()`
* [ ] Finish blog post
* [ ] Tweet
* [ ] Add link to blog post in pkgdown news menu

### PR DESCRIPTION
**Pull request by dpprdan**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Type: Package
 Package: opencage
 Title: Geocode with the OpenCage API
-Version: 0.1.4.9007
+Version: 0.2.0
 Authors@R: c(
-    person("Maëlle", "Salmon", email = "maelle.salmon@yahoo.se", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-2815-0399")), 
-    person("Daniel", "Possenriede", role = "aut", comment = c(ORCID = "0000-0002-6738-9845")),
+    person("Daniel", "Possenriede", email = "possenriede+r@gmail.com", role = c("aut", "cre"), comment = c(ORCID = "0000-0002-6738-9845")),
```
Comment (maelle) [inline DESCRIPTION:6]: :tada:

**Comment by freyfogle** [comment]
Tears of joy are running down my face

File: DESCRIPTION (hunk 1 above)
Comment (jessesadler) [reply DESCRIPTION:6]: Very cool

**Comment by dpprdan** [comment]
> Thanks, we see:
Found the following (possibly) invalid URLs:
	URL: c (moved to https://opencagedata.com/guides/how-to-protect-your-api-key)
	From: inst/doc/opencage.html
	Status: 200
	Message: OK 
Please change http --> https, add trailing slashes, or follow moved content as appropriate. Please fix and resubmit.

This wasn't caught by {[urlchecker](https://github.com/r-lib/urlchecker)},  because its `url_check()`s in Rmds and READMEs rely on Pandoc being on the PATH (https://github.com/r-lib/urlchecker/issues/10). 

Use `withr::with_path(Sys.getenv("RSTUDIO_PANDOC"), urlchecker::url_check())` until this is fixed.

**Comment by dpprdan** [comment]
CRAN says:
> package opencage_0.2.1.tar.gz has been auto-processed.
We are waiting for confirmation from the old maintainer address now.

**Pull request merged**